### DISCUSSION TITLE
Is it possible to open Monaco Editor without FileResource?

### DISCUSSION THREAD

**Discussion by minyoungyang** [eclipse-theia/theia#7904] [General] [answered]
Is it possible to open Monaco Editor without FileResource?
For example, VSCode provides 'New File' as below.
![image](https://user-images.githubusercontent.com/48236160/82996284-af245880-a03f-11ea-89a6-771f30cf20b4.png)

This seems to have opened Monaco Editor without FileResource.

**Comment by akosyakov** [comment, marked as answer]
You should be able to achieve it by using `untitled` scheme. There is [createUntitledURI](https://github.com/eclipse-theia/theia/blob/f56cb7a7648012142099d022180c79c289947e2b/packages/plugin-ext/src/main/browser/editor/untitled-resource.ts#L129) to create such URI.

**Comment by minyoungyang** [comment]
@akosyakov 
Thanks for your reply. Your reply is what I wanted.
